cmd/kube-controller-manager/app/options/nodelifecyclecontroller.go

```diff
@@ -45,7 +45,8 @@ func (o *NodeLifecycleControllerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.Int32Var(&o.LargeClusterSizeThreshold, "large-cluster-size-threshold", 50, "Number of nodes from which NodeController treats the cluster as large for the eviction logic purposes. --secondary-node-eviction-rate is implicitly overridden to 0 for clusters this size or smaller.")
 	fs.Float32Var(&o.UnhealthyZoneThreshold, "unhealthy-zone-threshold", 0.55, "Fraction of Nodes in a zone which needs to be not Ready (minimum 3) for zone to be treated as unhealthy. ")
 	fs.BoolVar(&o.EnableTaintManager, "enable-taint-manager", o.EnableTaintManager, "If set to true enables NoExecute Taints and will evict all not-tolerating Pod running on Nodes tainted with this kind of Taints.")
-	fs.MarkDeprecated("enable-taint-manager", "This flag is deprecated and it will be removed in 1.26. The taint-manager is enabled by default and will remain implicitly enabled once this flag is removed.")
+	fs.MarkDeprecated("enable-taint-manager", "This flag is deprecated and it will be removed in 1.27. The taint-manager is enabled by default and will remain implicitly enabled once this flag is removed.")
```

Comment:
We can still remove this in 1.26 and mark pod-eviction-timeout as deprecated in the same release.

I would just do that in the same PR.

Reply:
But it seems a little weird for we can set the pod-eviction-timeout but actually it doesn't work. 🤔 
Or `Deprecated == Can't use` ?

Reply:
Then what does `deprecation` works for here?

Reply:
It has 2 effects if you still have `--pod-eviction-timeout` in the command line arguments:

- it will continue to work (the binary won't crash on startup)
- you will get a warning in the logs

Reply:
Got it. Thanks for explaining. I'll update in the previous PR.